apps/hubble-stats/app/layout.tsx

```diff
@@ -77,6 +85,35 @@ export default function RootLayout({
     Icon: DocumentationIcon,
   };
 
+  const pathname = usePathname();
+
+  const breadCrumbs = useMemo(() => {
+    const parts = pathname.split('/');
+    const activeItem = parts[parts.length - 1];
+
+    const breadCrumbItems: Breadcrumb[] = [
+      {
+        label: 'Hubble Overview',
+        isLast: activeItem !== '' ? false : true,
+        icon: (
+          <ContrastLine className={activeItem !== '' ? 'fill-mono-120' : ''} />
+        ),
+        href: '/',
+      },
+    ];
+
+    if (activeItem !== '') {
+      breadCrumbItems.push({
+        label: activeItem,
+        isLast: true,
+        icon: <CoinIcon />,
+        href: '',
+      });
+    }
+
+    return breadCrumbItems;
+  }, [pathname]);
+
```

Comment:
This component should be the "server component" (or async component to fetch TVL and volume), passed fetched data into the `Header` component to render. Move this logic to the `Header` component, extract it into a React hook, and use it inside the `Header` client component.

Reply:
Also, consider moving static variables (`items` and `footer` in this case) outside of the component.